### ISSUE TITLE
"New Epic" does not create new issues

### ISSUE THREAD

**Issue by shlomi-nx** [codetree/feedback#363]
For the past few days, trying the create a new epic simply keeps saying "saving" but nothing is created. 

I tried deleting cache, logout/login, different browsers (chrome/safari).

Any workarounds for the moment?

**Comment by shlomi-nx** [comment]
Looking at the console, I see

```
application-45417344ff86fe582ba3deac9a84b8da.js:6 Uncaught TypeError: Cannot read property 'added_labels' of undefined
    at h.submit (application-45417344ff86fe582ba3deac9a84b8da.js:6)
    at h.<anonymous> (frameworks-39816068afde52aba78cfed201521fa4.js:224)
    at Array.forEach (<anonymous>)
    at h.<anonymous> (frameworks-39816068afde52aba78cfed201521fa4.js:224)
    at HTMLDivElement.dispatch (frameworks-39816068afde52aba78cfed201521fa4.js:26)
    at HTMLDivElement.m.handle (frameworks-39816068afde52aba78cfed201521fa4.js:25)
    at HTMLDivElement.C.t.___hb.t.___hb (frameworks-39816068afde52aba78cfed201521fa4.js:342)
submit @ application-45417344ff86fe582ba3deac9a84b8da.js:6
(anonymous) @ frameworks-39816068afde52aba78cfed201521fa4.js:224
(anonymous) @ frameworks-39816068afde52aba78cfed201521fa4.js:224
dispatch @ frameworks-39816068afde52aba78cfed201521fa4.js:26
m.handle @ frameworks-39816068afde52aba78cfed201521fa4.js:25
C.t.___hb.t.___hb @ frameworks-39816068afde52aba78cfed201521fa4.js:342
```

**Comment by shlomi-nx** [comment]
Which takes to this:
```javascript
this.submit = function() {
        if (this.clearErrors(),
        this.isValid()) {
            this.disable(),
            this.$node.attr("data-state", "submitting");
            var e = {
                projectId: this.projectId(),
                filter: this.currentParams(),
                params: this.serializeForm()
            }
              , t = this.select("stageModalSelector").filter(":checked").data("transform");
            t.added_labels && (e.params.labels = e.params.labels.concat(t.added_labels));        /// This line is underlined as the error
            var i = "epic" === this.issueType() ? "uiCreateEpicRequested" : "uiCreateIssueRequested";
            return this.trigger(i, e)
        }
        return this.displayErrors(this.errors)
    }
```

**Comment by ac21** [comment]
Hey @shlomi-nx - thanks for raising attention to this issue.  Working on it now and hoping to have a fix by EOD tomorrow

**Comment by shlomi-nx** [comment]
If anyone else is stuck here, for a workaround add some random tags and create the epic. Afterwards remove the tags. Worked for me.

**Comment by ac21** [comment]
links to codetree/codetree#2367

**Comment by ac21** [comment]
resolved by codetree/codetree#2367
